concepts/type-conversion/about.md

```diff
@@ -1,10 +1,272 @@
+In JavaScript, values may be of different types. Changing the type of a variable can be done by explicit _type conversion_.
+Besides that, JavaScript also performs _type coercion_ (implicit type conversion) when the context requires it.
 
-Generally, type coercion is avoided because it is averse to unexpected errors. Instead, opt for practices which allow for explicit type conversion.
+## Type Conversion
 
-An overview of these terms can be found on _MDN Web Docs_:
+JavaScript does not have a construct to cast into a (different) type like many other languages but there are built-in helpers that can be used instead.
+Most notably, the global objects `Boolean`, `Number` and `String` can be used as functions to convert a value to the respective type.
 
-- [Type Conversion](https://developer.mozilla.org/en-US/docs/Glossary/Type_Conversion)
-- [Type Coercion](https://developer.mozilla.org/en-US/docs/Glossary/Type_coercion)
+### Converting to a Boolean (Truthy/Falsy Values)
+
+With `Boolean(value)` you can convert any value to a boolean.
+How does that work?
+
+There is a fixed set of values, so called _falsy_ values, that convert to `false`.
+Most importantly, `false`, `0`, empty string, `null`, `undefined` and `NaN` are falsy.
+The [MDN article on "Falsy"][mdn-falsy] shows the complete list.
+
+For all other values, `Boolean` returns `true`.
+These values are called _truthy_.
+
+```javascript
+Boolean(-1);
+// => true
+
+Boolean(0);
+// => false
+
+Boolean(' ');
+// => true
+
+Boolean('');
+// => false
+```
+
+Note that because of the described rules, the following values are truthy in JavaScript although they might count as falsy in other languages.
+
+```javascript
+Boolean('0');
+// => true
+
+Boolean('false');
+// => true
+
+Boolean([]);
+// => true
+
+Boolean({});
+// => true
+```
+
+### Converting to a Number
+
+`Number(value)` can be used to convert a value to a number.
+Whitespaces at the beginning and the end of a string are ignored and an empty string is converted to `0`.
+If you try to convert a non-primitive value or a string that does not represent a number, **no** error will be thrown.
+Instead, the result is `NaN` ([Not-A-Number][mdn-nan]).
+
+```javascript
+Number('  -12.34  ');
+// => -12.34
+
+Number('1,2');
+// => NaN
+
+Number('1 2');
+// => NaN
+
+Number('');
+// => 0
+
+Number('10e3');
+// => 10000
+
+Number({ key: '123' });
+// => NaN
+```
+
+Below you can see what `Number` returns for other primitive values.
+
+```javascript
+Number(true);
+// => 1
+
+Number(false);
+// => 0
+
+Number(null);
+// => 0
+
+Number(undefined);
+// => NaN
+```
+
+JavaScript also provides the functions `parseInt` and `parseFloat`.
+They apply laxer rules as to which strings can be converted to a number.
+Because of that, `Number` should be preferred as conversion function to avoid unexpected outcomes.
```

Comment:
😇 I learned something today... I have so many codebases to fix. I had so many bugs caused by forgetting the second (base) argument to `parseInt`...